crates/cast/src/main.rs

```diff
@@ -214,6 +214,8 @@ async fn main() -> Result<()> {
                     &cli.path_to_scarb_toml,
                     &provider,
                     &add,
+                    cli.keystore,
+                    cli.account.map(Utf8PathBuf::from),
```

Comment:
this (account) is a part of CastConfig, right? maybe we should not map it here, but rather in the add function itself, when it's needed, and it could be taken from the config?